architecture/kubernetes_infrastructure.adoc

```diff
@@ -9,43 +9,41 @@
 
 toc::[]
 
-Kubernetes is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenance, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications. 
+== Overview
+http://kubernetes.io[Kubernetes] is a system for managing containerized applications across a set of containers or hosts, providing deployment, maintenance, and application-scaling mechanisms. Docker is used to package, instantiate, and run containerized applications.
 
-A Kubernetes cluster consists of a master and a set of nodes. 
+A Kubernetes cluster consists of a master and a set of nodes.
 
 == Master
-
-The Kubernetes master is the host or hosts that contain the master components (API Server, Controller Manger Server, etcd, etc.).
+The master is the host or hosts that contain the master components, including the API server, controller manager server, and [sysitem]#etcd#. The master manages link:#node[nodes] in its Kubernetes cluster and schedules link:kubernetes_model.html#pod[pods] to run on nodes.
 
 === API Server
+The Kubernetes API server serves the main API, and validates and configures the data for pods, services, and replication controllers. The Kubernetes API server also assigns pods to nodes and synchronizes pod information with service configuration.
 
-The Kubernetes API server serves the main API, and validates and configures the data for pods, services, and replication controllers. The Kuberbetes API server also assigns pods to nodes and synchronizes pod information with service configuration. 
-
-=== etcd
-
-Persistent master state is stored in etcd. Other components watch etcd for changes and eventually bring themselves into the desired state described in etcd.
+=== [sysitem]#etcd#
+Persistent master state is stored in [sysitem]#etcd#. Other components watch [sysitem]#etcd# for changes and eventually bring themselves into the desired state described in [sysitem]#etcd#.
```

Comment:
I'm not sure if I understand this. Can we elaborate and clarify this from the customer's perspective?